data/en/1_0102.json

```diff
@@ -208,7 +208,10 @@
                         "q_code": "21",
                         "type": "Currency",
                         "currency": "GBP",
-                        "decimal_places": 2
+                        "decimal_places": 2,
+                        "max_value": {
```

Comment:
Which error message will it display?

Reply:
`Enter a number less than or equal to £100.00.`

Reply:
Once updated with master will read 'Enter an answer less than or equal to £100.00.`